update-po-files.sh

```diff
@@ -10,7 +10,9 @@
 #
 #####################################
 
-LANG=C
+LC_ALL=C
+
+export LC_ALL
```

Comment:
Why is this not a one liner?

Reply:
Why it should be a one liner?

Reply:
Sorry, I'm a traditionalist, and this doesn't work as an oneliner with ancient shells. Then again, the entire script needs bash extensions.

Reply:
Actually at the top of it there is the classic shebang #!/bin/bash -e but I agree with you, I am a traditionalist too! :-)